audio/out/ao.c

```diff
@@ -506,6 +507,16 @@ struct ao_device_list *ao_hotplug_get_device_list(struct ao_hotplug *hp)
     MP_TARRAY_APPEND(list, list->devices, list->num_devices,
         (struct ao_device_desc){"auto", "Autoselect device"});
 
+    if (!hp->ao && playback_ao && playback_ao->driver->hotplug_init) {
```

Comment:
I'd add a short comment here explaining that we prefer the currently playing ao for hotplug and why